src/components/Whiteboard/ElementBehaviors/Resizable/ResizeHandle.tsx

```diff
@@ -17,101 +17,127 @@
-          0,
-          containerWidth + 2 * selectionBorderPadding - handleWidth,
-        ),
-        height: handleWidth,
-        cursor: 'ns-resize',
-      };
-    case 'topRight':
-      return {
-        x: containerWidth - handleWidth / 2 + selectionBorderPadding,
-        y: -handleWidth / 2 - selectionBorderPadding,
-        width: handleWidth,
-        height: handleWidth,
-        cursor: 'ne-resize',
-      };
-    case 'right':
-      return {
-        x: containerWidth + selectionBorderPadding - handleWidth / 2,
-        y: handleWidth / 2 - selectionBorderPadding,
-        width: handleWidth,
-        height: Math.max(
-          0,
-          containerHeight + 2 * selectionBorderPadding - handleWidth,
-        ),
-        cursor: 'ew-resize',
-      };
-    case 'bottomRight':
-      return {
-        x: containerWidth - handleWidth / 2 + selectionBorderPadding,
-        y: containerHeight - handleWidth / 2 + selectionBorderPadding,
-        width: handleWidth,
-        height: handleWidth,
-        cursor: 'se-resize',
-      };
-    case 'bottom':
-      return {
-        x: handleWidth / 2 - selectionBorderPadding,
-        y: containerHeight - handleWidth / 2 + selectionBorderPadding,
-        width: Math.max(
-          0,
-          containerWidth + 2 * selectionBorderPadding - handleWidth,
-        ),
-        height: handleWidth,
-        cursor: 'ns-resize',
-      };
-    case 'bottomLeft':
-      return {
-        x: -handleWidth / 2 - selectionBorderPadding,
-        y: containerHeight - handleWidth / 2 + selectionBorderPadding,
-        width: handleWidth,
-        height: handleWidth,
-        cursor: 'sw-resize',
-      };
-    case 'left':
-      return {
-        x: -handleWidth / 2 - selectionBorderPadding,
-        y: handleWidth / 2 - selectionBorderPadding,
-        width: handleWidth,
-        height: Math.max(
-          0,
-          containerHeight + 2 * selectionBorderPadding - handleWidth,
-        ),
-        cursor: 'ew-resize',
-      };
-    case 'topLeft':
-      return {
-        x: -handleWidth / 2 - selectionBorderPadding,
-        y: -handleWidth / 2 - selectionBorderPadding,
-        width: handleWidth,
-        height: handleWidth,
-        cursor: 'nw-resize',
-      };
+  if (elementKind === 'line') {
+    const { handlePositionX = 0, handlePositionY = 0 } = params;
+
+    switch (handlePosition) {
+      case 'start':
+        return {
+          x: handlePositionX + handleWidth / -2,
+          y: handlePositionY + handleWidth / -2,
+          width: handleWidth,
+          height: handleWidth,
+          cursor: 'default',
+        };
+      case 'end':
+        return {
+          x: handlePositionX + handleWidth / -2,
+          y: handlePositionY + handleWidth / -2,
+          width: handleWidth,
+          height: handleWidth,
+          cursor: 'default',
+        };
+    }
+  } else {
+    const { handlePosition, containerWidth = 0, containerHeight = 0 } = params;
```

Comment:
`handlePosition` is already defined here, no need to assign defaults to `containerWidth`, `containerHeight`

Reply:
`handlePosition` could be removed, it is already defined in line 34